src/PrettyVersions.php

```diff
@@ -2,14 +2,16 @@
 
 namespace Jean85;
 
-use PackageVersions\Versions;
+use Composer\InstalledVersions;
 
 class PrettyVersions
 {
     const SHORT_COMMIT_LENGTH = 7;
 
     public static function getVersion(string $packageName): Version
     {
-        return new Version($packageName, Versions::getVersion($packageName));
+        $version = InstalledVersions::getPrettyVersion($packageName) . '@' . InstalledVersions::getReference($packageName);
```

Comment:
Have you read the docs there? This might return `null.'@'.null` for replaced/provided packages.

Reply:
Handling this in #16